kb/2022-03-10/multiple_nics_vm_connectivity.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Multiple NICs VM Connectivity
+descripion: How to deal VMs with multiple NICs in Harvester
+slug: multiple-nics-vm-connectivity
+authors:
+  - name: Date Huang
+    title: Software Engineer
+    url: https://github.com/tjjh89017
+    image_url: https://github.com/tjjh89017.png
+tags: [vm, network]
+hide_table_of_contents: false
+---
+
+# Multiple NICs VM Connectivity
+
+## What is the default behavior of a VM with multiple NICs
+
+In some scenarios, you'll setup two or more NICs in your VM to serve different networking purposes. If all networks are setup by default with DHCP, you might get random connectivity issues. And while it might get fixed after rebooting the VM, it still will lose connection randomly after some period.
+
```

Comment:
not sure how to link it will be better.

Reply:
how about [some scenarios](https://github.com/harvester/harvester/issues/1059)?

Reply:
got it.

Reply:
```suggestion
In [some scenarios](https://github.com/harvester/harvester/issues/1059), you'll setup two or more NICs in your VM to serve different networking purposes. If all networks are setup by default with DHCP, you might get random connectivity issues. And while it might get fixed after rebooting the VM, it still will lose connection randomly after some period.

```

Reply:
seems this is not commited?

Reply:
ok fixed it!